plot-doc/plot/scribblings/contracts.scrbl

```diff
@@ -50,6 +50,32 @@ The @racket['auto] anchor is only valid for placement of text labels, for all
 other use cases, the @racket['auto] anchor is always the same as
 @racket['bottom-left].}
 
+@defthing[legend-anchor/c contract? #:value (or/c anchor/c
+                                                  (one-of/c 'no-legend
+                                                            'outside-top-left    'outside-top    'outside-top-right
+                                                            'outside-left-top    'outside-left   'outside-left-bottom
+                                                            'outside-right-top   'outside-right  'outside-right-bottom
+                                                            'outside-bottom-left 'outside-bottom 'outside-bottom-right
+                                                            'outside-global-top))]{
+The contract for the @(racket plot-legend-anchor) parameter.
```

Comment:
Can you use the following text for the documentation (it documents the values in the logical order: inside, outside, no-legend and explains the purpose of auto)

```scribble
The contract for the @racket[plot-legend-anchor] parameter and the
@racket[#:legend-anchor] parameters for the various plot procedures.

When @racket[legend-anchor] is one of the symbols from @racket[anchor/c], the
legend will be placed inside the plot area.

@racket[legend-anchor/c] values which start with "outside" will place the
legend outside the plot area, for 2D plots the legend will be aligned with the
plot area, while for 3D plots the legend will be relative to the overall
@racket[plot-width] and @racket[plot-height].

The @racket['outside-global-top] value will place the legend the plot-area,
centered on the complete @racket[plot-width]. For 3D plots there is no
difference between this value and @racket['outside-top].

The value @racket['no-legend], will omit the legend from the plot, the legend
will also be omitted if none of the renderers have a @racket[#:label]
specified, regardless of the value used for @racket[#:legend-anchor].

The value @racket['auto], will place the legend in the top-left corner of the
plot area, this is not usefull for plot legends, this ancor values is used for
renderers such as @racket[point-label].
````